rest/blip_sync.go

```diff
@@ -175,6 +175,16 @@ func (bh *blipHandler) handleSetCheckpoint(rq *blip.Message) error {
 // Received a "subChanges" subscription request
 func (bh *blipHandler) handleSubscribeToChanges(rq *blip.Message) error {
 	var since db.SequenceID
+
+	// Depeding on the db sequence type, use correct zero sequence for since value
+	// TODO #1: Add unit tests or spin off separate ticket
+	// TODO #2: Doesn't this need a valid SeqType field?  Seems to work without it
+	if bh.db.SequenceType == db.IntSequenceType {
+		since = db.SequenceID{Seq: 0}
+	} else {
+		since = db.SequenceID{Clock: base.NewSequenceClockImpl()}
+	}
+
```

Comment:
It feels like we should pull the code: https://github.com/couchbase/sync_gateway/blob/4c6bd7044a31a82fcf8a44f02df9a52ecdd07ad5/rest/changes_api.go#L800-L804

into a helper method that could be called from changes_api and here - what do you think?